docs/docsite/rst/community/development_process.rst

```diff
@@ -23,6 +23,62 @@ Ansible accepts code via **pull requests** ("PRs" for short). GitHub provides a
 
 Because Ansible receives many pull requests, we use an automated process to help us through the process of reviewing and merging pull requests. That process is managed by **Ansibullbot**.
 
+Backport Pull Request Process
+-----------------------------
+
+After the pull request submitted to Ansible for the ``devel`` branch is
+accepted and merged, the following instructions will help you create a
+pull request to backport the change to a previous stable branch.
+
+.. note::
+
+    These instructions assume that ``stable-2.5`` is the targeted release
+    branch for the backport.
+
+.. note::
+
+    These instructions assume that ``https://github.com/ansible/ansible.git``
+    is configured as a ``git remote`` named ``upstream``. If you do not use
+    a ``git remote`` named ``upstream`` adjust the instructions accordingly.
+
+#. Prepare your devel, stable, and feature branches:
+
+   ::
+
+       git checkout devel
+       git --fetch upstream
+       git merge upstream/devel
+       git checkout stable-2.5
+       git pull
+       git checkout -b cherry-pick/2.5/[PR_NUMBER_FROM_DEVEL]
+
```

Comment:
not sure why this step is needed, since the instructions assume devel has the commit as seen below

Reply:
Not sure I understand this comment.  The idea is that we are creating a feature branch from a stable branch, cherry picking into the stable feature branch from devel, so we can push and create a PR.

Reply:
ah, checkout is not really best way, it seems you just want -m (rename)